modules/nw-ovn-kubernetes-readiness-probes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-project-readiness-probes_{context}"]
+= OVN-Kubernetes readiness probes
```

Comment:
This title looks like it's for a concept module.

Reply:
Rewritten correctly to reflect that this is a procedure my bad